SuperBuild.cmake

```diff
@@ -118,7 +118,14 @@ if(Slicer_BUILD_DICOM_SUPPORT)
 endif()
 
 if(Slicer_BUILD_DICOM_SUPPORT AND Slicer_USE_PYTHONQT_WITH_OPENSSL)
+  list(APPEND Slicer_DEPENDENCIES python-pydicom1)
   list(APPEND Slicer_DEPENDENCIES python-pydicom)
+  list(APPEND Slicer_DEPENDENCIES python-certifi)
+  list(APPEND Slicer_DEPENDENCIES python-idna)
+  list(APPEND Slicer_DEPENDENCIES python-chardet)
+  list(APPEND Slicer_DEPENDENCIES python-urllib3)
+  list(APPEND Slicer_DEPENDENCIES python-requests)
+  list(APPEND Slicer_DEPENDENCIES python-dicomweb-client)
```

Comment:
Could you dispatch the indirect dependencies in their respective external project ? That way only `python-dicomweb-client` would be listed here.

Reply:
Thanks for the review Jc 👍 

What do you mean by dispatch?  I didn't find anyplace where these dependencies were listed, I only discovered them by trying to import the package.

Reply:
The idea is to list the direct dependency here: https://github.com/Slicer/Slicer/pull/1081/files#diff-248ac0f903bd7e21b07e0b618e2bba3cR4

For pydicom, they can be extracted from here: https://github.com/clindatsci/dicomweb-client/blob/bfbaff8cfdc74d27fb30b9ea8cc462d1e317135d/setup.py#L58-L63

Reply:
Ok, I see what you mean, yes, that would be cleaner.